### DISCUSSION TITLE
[BUG]

### DISCUSSION THREAD

**Discussion by Laconiss** [Chia-Network/chia-blockchain#3164] [General]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

- OS: [e.g. Linux]
- OS Version/Flavor: [e.g. CentOS 7.2]
- CPU: [e.g. Intel Xeon 8175M]

**Additional context**
![IMG_20210429_103637](https://user-images.githubusercontent.com/81981907/116528767-7db1ae00-a8dc-11eb-90c8-e2614a5c6be8.jpg)
 Windows cant farmer all time

**Comment by Laconiss** [comment]
Look date...is last day

**Comment by Laconiss** [comment]
Have people help me??? Plz

**Comment by Cast39** [comment]
Syncing will take some time, but to speed the process a up bit you can follow this guide:
[https://github.com/Chia-Network/chia-blockchain/wiki/Port-8444](url)

**Comment by Laconiss** [comment]
> 同步将需要一些时间，但是要加快处理速度，您可以按照以下指南进行操作：[https](url) :
> [//github.com/Chia-Network/chia-blockchain/wiki/Port-8444](url)

Full node status is synced..wallet too..just farm last attempted proof date not move..plz help me

**Comment by sargonas** [comment]
Tagging this issue for migration to a discussion. Apologies in advance, but you will be best served by moving this over there and out of the issue tracker, where the community can better assist you. (Ideally the issue tracker should be used to focus on bugs and development problems that need to be addressed, with general user support handled in Discussions and Keybase)

^MigrateToDiscussion
